request.coffee

```diff
@@ -2,17 +2,55 @@ _ = require 'lodash'
 request = require 'request'
 Promise = require 'bluebird'
 PinejsClientCore = require './core'
+BluebirdLRU = require 'bluebird-lru-cache'
 
 request = Promise.promisify(request)
 
+validParams = ['cache']
 module.exports = class PinejsClientRequest extends PinejsClientCore(_, Promise)
+	constructor: (params, backendParams) ->
```

Comment:
They could be used by the angular backend, but their use is up to the discretion of the backend - angular doesn't have any specific params that it needs yet (the browser deals with caching for the angular version)